task/net/src/bsp/grapefruit.rs

```diff
@@ -0,0 +1,157 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#[cfg(not(feature = "ksz8463"))]
+compile_error!("this BSP requires the ksz8463 feature");
```

Comment:
this BSP is only enabled if the `"grapefruit"` feature flag is enabled, which itself enables the `"ksz8463"` feature, right? so this compile error should, AFAICT, never occur?

Reply:
Yes, this may also be copypasta
```console
$ rg "this BSP requires"
task/net/src/bsp/sidecar_bcd.rs
11:compile_error!("this BSP requires the ksz8463 and mgmt features");

task/net/src/bsp/grapefruit.rs
6:compile_error!("this BSP requires the ksz8463 feature");

task/net/src/bsp/gimlet_bcdef.rs
6:compile_error!("this BSP requires the ksz8463 and mgmt features");

task/net/src/bsp/gimletlet_mgmt.rs
6:compile_error!("this BSP requires the ksz8463 and mgmt features");

task/net/src/bsp/psc_bc.rs
6:compile_error!("this BSP requires the ksz8463 and mgmt features");

task/net/src/bsp/psc_a.rs
6:compile_error!("this BSP requires the ksz8463 and mgmt features");

task/net/src/bsp/gimletlet_nic.rs
6:compile_error!("this BSP requires the ksz8463 feature");
```

I'm fine deleting it and letting things fail naturally if someone messes it up.